dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListener.cpp

```diff
@@ -1,12 +1,16 @@
-﻿#include "pch.h"
+﻿#pragma once
+
+#include "pch.h"
 #include "../PushNotificationUtility.h"
 
 HRESULT NotificationListener::RuntimeClassInitialize(
     std::shared_ptr<ForegroundSinkManager> foregroundSinkManager,
+    std::shared_ptr<ToastRegistrationManager> toastRegistrationManager,
     std::wstring appId,
```

Comment:
Can the wstrings be referenced?

Reply:
Can you explain a little about what this means?

Reply:
I mean, expect a wstring reference as a parameter instead of copying: 
```suggestion
    std::wstring& appId,
```